src/schemas/user.py

```diff
@@ -0,0 +1,39 @@
+from datetime import date, datetime
+from typing import Optional
+
+from pydantic import BaseModel, Field, validator
+
+
+class UserSchema(BaseModel):
+    """
+    Pydantic model that will we used to send data to and from routes.
+    Has attributes and validations as of DB.
+    """
+
+    id: Optional[int] = None
+    email: str = Field(min_length=0, max_length=32)
+    username: str = Field(min_length=0, max_length=32)
+    password: str = Field(min_length=0, max_length=8)
+    first_name: str = Field(min_length=0, max_length=32)
+    last_name: str = Field(min_length=0, max_length=32)
+    date_of_joining: datetime = Field()
```

Comment:
Shouldn't date of joining be optional? If not passed then can be set to current date?

Reply:
Yes but this also allows the user to provide an incorrect date of joining. We can have timezone support in datetime fields so time difference shouldn't be a problem imo